src/parquet/util/rle-test.cc

```diff
@@ -317,15 +344,27 @@ TEST(BitRle, Flush) {
 
 // Test some random sequences.
 TEST(BitRle, Random) {
-  int iters = 0;
-  while (iters < 1000) {
-    srand(iters++);
-    if (iters % 10000 == 0) LOG(ERROR) << "Seed: " << iters;
-    vector<int> values;
+  size_t niters = 50;
+  size_t ngroups = 1000;
+  size_t max_group_size = 16;
+  vector<int> values(ngroups + max_group_size);
+
+  // prng setup
+  std::random_device rd;
+  std::uniform_int_distribution<int> dist(1, std::numeric_limits<int>::max());
```

Comment:
A small optimization: `std::uniform_int_distribution<int> dist(1,20);`
and change line 366 to
`int group_size = dist(gen);`